src/main/java/com/miraisolutions/xlconnect/integration/r/RWorkbookWrapper.java

```diff
@@ -234,9 +234,9 @@ public boolean isSheetVeryHidden(String sheetName) {
         return workbook.isSheetVeryHidden(sheetName);
     }
 
-    public void addImage(String filename, String name, boolean originalSize)
-            throws FileNotFoundException, IOException {
-        workbook.addImage(filename, name, originalSize);
+    public void addImage(String filename, String name, String worksheetScope, boolean originalSize)
```

Comment:
If I get this right, in these wrappers we reflect the order of arguments as in the R functions, where `worksheetScope` should be added *last* for backwards compatibility. This is not the case on the R side, see https://github.com/miraisolutions/xlconnect/pull/146/commits/d377ac858a410de93dd70216b721e053f1951617#diff-b9752c2d5f2b7b47ba69c27042a662001c0a9522ce5aac09e6783e6e1ecf68d7, where the same changes applies

```suggestion
    public void addImage(String filename, String name, boolean originalSize, String worksheetScope)
```

Reply:
Thanks, I had forgotten why I did that in the other ones :sweat_smile: